certbot/tests/test_fake_marathon.py

```diff
@@ -0,0 +1,189 @@
+import json
+
+import treq
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.protocols.loopback import _LoopbackAddress
+from twisted.web.server import Site
+
+from certbot.tests.fake_marathon import FakeMarathon, FakeMarathonAPI
+from certbot.tests.helpers import FakeServerAgent, TestCase
+from certbot.tests.matchers import IsJsonResponseWithCode
+
+from testtools.matchers import Equals
+
+from txfake import FakeServer
+
+from uritools import uricompose
```

Comment:
From PEP-8:

    Imports should be grouped in the following order:

    1. standard library imports
    2. related third party imports
    3. local application/library specific imports

    You should put a blank line between each group of imports.